process/interceptors/disabledWhiteListDataVerifier.go

```diff
@@ -0,0 +1,32 @@
+package interceptors
+
+import (
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+type disabledWhiteListVerifier struct {
+}
+
+// NewDisabledWhiteListDataVerifier returns a default data verifier
+func NewDisabledWhiteListDataVerifier(_ storage.Cacher) (*disabledWhiteListVerifier, error) {
+	return &disabledWhiteListVerifier{}, nil
+}
+
+// IsWhiteListed return true if intercepted data is accepted
+func (w *disabledWhiteListVerifier) IsWhiteListed(interceptedData process.InterceptedData) bool {
+	return false
+}
+
+// Add ads all the list to the cache
```

Comment:
adds*
We usually have comments in disabled components like: `Add does nothing`

Reply:
fixed the typo